### PR TITLE
Update getting-started.asciidoc

### PR DESCRIPTION
**Pull request by doggy8088**
Bug code fixed

### CODE REVIEW

**Comment by doggy8088** [comment]
@cla-checker-service just signed.

**Comment by russcam** [comment]
@doggy8088 thanks for the PR. Could you please update https://github.com/elastic/elasticsearch-net/blob/7.x/tests/Tests/ClientConcepts/LowLevel/GettingStarted.doc.cs, the file from which the asciidoc is generated? Thanks!

**Comment by doggy8088** [comment]
@russcam Yes, already done.

**Comment by doggy8088** [comment]
@russcam Should I remove the commit for `docs/client-concepts/low-level/getting-started.asciidoc` ?

**Comment by russcam** [comment]
_Ideally_, `docs/client-concepts/low-level/getting-started.asciidoc` is committed too, which can be generated from a change to https://github.com/elastic/elasticsearch-net/blob/7.x/tests/Tests/ClientConcepts/LowLevel/GettingStarted.doc.cs by running

```
./build documentation
```
from the root of the solution

File: tests/Tests/ClientConcepts/LowLevel/GettingStarted.doc.cs [hunk 1]
```diff
@@ -192,8 +192,9 @@ public void SearchingWithAnonymousTypes()
 				{
 					match = new
 					{
-						field = "firstName",
-						query = "Martijn"
+						firstName = new {
```
Comment (russcam) [inline tests/Tests/ClientConcepts/LowLevel/GettingStarted.doc.cs:195]:
```suggestion
						firstName = new
						{
```

File: docs/client-concepts/low-level/getting-started.asciidoc [hunk 2]
```diff
@@ -176,8 +176,9 @@ var searchResponse = lowlevelClient.Search<StringResponse>("people", PostData.Se
     {
         match = new
         {
-            field = "firstName",
-            query = "Martijn"
+            firstName = new {
```
Comment (russcam) [inline docs/client-concepts/low-level/getting-started.asciidoc:179]:
```suggestion
            firstName = new
            {
```

**Pull request merged**